scripts/preprocess/custom_projection.js

```diff
@@ -0,0 +1,279 @@
+var d3 = require('d3');
+
+var epsilon = 1e-6;
+
+// The projections must have mutually exclusive clip regions on the sphere,
+// as this will avoid emitting interleaving lines and polygons.
+function multiplex$1(streams) {
+  var n = streams.length;
+  return {
+    point: function(x, y) { var i = -1; while (++i < n) streams[i].point(x, y); },
+    sphere: function() { var i = -1; while (++i < n) streams[i].sphere(); },
+    lineStart: function() { var i = -1; while (++i < n) streams[i].lineStart(); },
+    lineEnd: function() { var i = -1; while (++i < n) streams[i].lineEnd(); },
+    polygonStart: function() { var i = -1; while (++i < n) streams[i].polygonStart(); },
+    polygonEnd: function() { var i = -1; while (++i < n) streams[i].polygonEnd(); }
+  };
+}
+
+var custom_projection = module.exports = {
```

Comment:
This second copy of custom_projection.js is necessary almost entirely because of this line (plus line 1, require(d3)). Here we're creating custom_projection as something that node.js will recognize as an exported and therefore `require`able entity, which is why we can call `require('custom_projection.js')` in process_counties_to_states.js (https://github.com/USGS-VIZLAB/water-use-15/pull/224/files#diff-f057c7e7b63321a9e7a46b1a9ca39d1aR23). 

In future vizzies, I think we should figure out how to make custom_projection.js into a module that can also be used directly in a webpage. It appears to be possible - see https://github.com/USGS-VIZLAB/water-use-15/issues/25#issuecomment-383757005 - but seems like a corner we ought to cut for now as our development time for water-use-15 is coming to a close.